### DISCUSSION TITLE
Installing and using

### DISCUSSION THREAD

**Discussion by RegressiaUK** [babybuddy/babybuddy#766] [General]
This project looks like a really useful thing, I would love to adapt it to use it for my own reasons I am incont and strange thing is part from the "pumping" and a few other options I have to monitor my sleep, eating and useage of diapers but I can't find any how to use or install anyway and your normal website is broken, the demo isn't but the site is.

Could you point me in to the right direction, my server is a MS 2019 server but from the basics i seen it's python if I am not mistaken anyway give us a helping hand thanks.

**Comment by billybonks** [comment]
HI, i deployed using the clever cloud guide it was pretty simple https://docs.baby-buddy.net/setup/deployment/#clever-cloud.

other then that there is guide for linux https://docs.baby-buddy.net/setup/deployment/#manual

i dont have much experience with MS Server

**Reply by RegressiaUK** [reply]
Thanks for your reply but I don't have a Linux system to hand and wont be having one and I wont be installing in any cloud as that goes out of the way of keeping personal data safe, Ya I know Windows Sucks but no at the same time it does not, and the things I run need windows and Linux can not run the things I have been running for the last 16 years :P

If anyone as means to get it working on windows it would be great what is fully local I don't think the server version of 2019 supports the wsl ecosystem but I guess that's something I might need to research.

**Reply by cdubz** [reply]
Yeah if you don't have Docker orchestration at hand you basically would need to look at the linked manual deployment steps and figure out how resolve for those dependencies on Windows. The documentation there is a tiny bit out of date but still correct.

The essentials are:

1. Python 3.10+
2. Pipenv for the virtual environment
3. Some means to serve requests from the virtual environment (the Linux steps use uwsgi and nginx for this purpose)
